src/Tizen.NUI.Components/Controls/Navigation/DialogPage.cs

```diff
@@ -80,9 +80,11 @@ protected override void Dispose(DisposeTypes type)
         }
 
         /// <summary>
-        /// Content of DialogPage. Content is added to Children automatically.
+        /// Content of DialogPage.
+        /// Content is used as dialog so Content is displayed above the dimmed scrim.
```

Comment:
i would add a coma:    ... as dialog, so ...

Reply:
Thank you for your comment.
It has been updated.